content/algorithms/porep/stacked_drg.md

```diff
@@ -234,7 +234,7 @@ TODO: link to filproofs/feistel
 
 > The Replication phase turns an *unsealed sector* into a *sealed sector* by first *generating a key*, then using the key to *encode the orignal data*.
 
-Before running the `Replicate` algorithm, the prover must ensure that the sector is correctly formatted with a valid "Storage Format" described in [Filecoin Client Data Processing](/missing_link) (currently `paddedfr32v1` is the required default).
+Before running the `Replicate` algorithm, the prover must ensure that the sector is correctly formatted with a valid "Storage Format" described in Filecoin Client Data Processing (currently `paddedfr32v1` is the required default).
```

Comment:
If we have nothing to link to here, we should re-word it to not imply that there is
```suggestion
Before running the `Replicate` algorithm, the prover must ensure that the sector is correctly formatted with a valid "Storage Format" (currently `paddedfr32v1` is the required default).
```

Reply:
i will update this !

Reply:
after `hugo: downloading modules ...` ends 😱